pages/about.html

```diff
@@ -5,4 +5,19 @@
 home: false
 ---
 
-<h2>About Content Goes Here</h2>
\ No newline at end of file
+<div class='quote-content-container'>
+  <div class='civic-info-container'>
+    <h1>What is Civic Tech?</h1>
+    <p>Civic tech bridges the gap between the civic space and technology. It’s utilizing technology practices like
+      design, engineering, data science, etc. to optimize the processes for civic organizations.</p>
+  </div>
+  <div class='quote-content'>
+    <div class='quote-paragraph-container'>
+      <p1>“...those of us who work in civic tech want public digital goods to be as good as the ones made by commercial
+        entities like Apple or Google—and we want public digital infrastructure to be as good, too.” </p1>
+      <br>
+      <p2>Cyd Harrell, A Civic Technologist’s Practice Guide</p2>
+    </div>
+    <img class="quote-icon" src="/images/about-page/quote.svg" alt="Quote Icon">
```

Comment:
Nice use of the alt attribute! However, this icon is entirely decorative. Can we set `alt=""`?